modules/tinymce/src/core/main/ts/html/ParserUtils.ts

```diff
@@ -52,11 +52,22 @@ const findClosestEditingHost = (scope: AstNode): Optional<AstNode> => {
   return Optional.from(editableNode);
 };
 
+const getAllDescendants = (scope: AstNode): AstNode[] => {
+  const collection: AstNode[] = [];
+
+  for (let node = scope.firstChild; node; node = node.walk()) {
```

Comment:
I think something like this would be more readable:

```suggestion
  for (let node = scope.firstChild; !Type.isNullable(node) ; node = node.walk()) {
```

Reply:
Yes I agree I copied the loop from another old function that had that truthy check. Well spotted.